common/fixation_migrations.go

```diff
@@ -0,0 +1,77 @@
+package common
+
+import (
+	"fmt"
+	"math"
+
+	sdk "github.com/cosmos/cosmos-sdk/types"
+	"github.com/lavanet/lava/common/types"
+)
+
+func prefixForErrors(from uint64) string {
+	return fmt.Sprintf("FixationStore: migration from version %d", from)
+}
+
+func (fs *FixationStore) MigrateVersion(ctx sdk.Context) (err error) {
+	from := fs.getVersion(ctx)
+	to := types.FixationVersion()
+
+	for from < to {
+		switch from {
+		case 1:
```

Comment:
why is the current version 1?

Reply:
getVersion() returns 1 if no version is set.